<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegexCheckClass</class>
 <widget class="QMainWindow" name="RegexCheckClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regex Check</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="inputRegex">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Expression:</string>
    </property>
   </widget>
   <widget class="QListView" name="resultList">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matches:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RegexCheck.qrc"/>
 </resources>
 <connections/>
</ui>
